deploy.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+
+from peering import AnnouncementController, ExperimentController
+
+
+def create_parser():
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "--announcement",
+        metavar="announcement_json",
+        type=str,
+        dest="announcement",
+        help="Deploy announcement via CLI",
+    )
+    group.add_argument(
+        "--experiment",
+        metavar="experiment_json",
+        type=str,
+        dest="experiment",
+        help="Deploy experiment via HTTP request",
+    )
+    parser.add_argument(
+        "--url",
+        required="--experiment" in sys.argv,
```

Comment:
I don't think this works. `sys.argv` may have `--experiment=path` instead, and that is completely valid. It seems we may want to create subcommands, and have `--url` be a subcommand of `--experiment`.

Reply:
subcommands seem to be what we want to use: https://docs.python.org/3/library/argparse.html#sub-commands